src/features/game/quiz/GameQuizConstants.ts

```diff
@@ -0,0 +1,51 @@
+import FontAssets from '../assets/FontAssets';
+import { screenSize } from '../commons/CommonConstants';
+import { BitmapFontStyle } from '../commons/CommonTypes';
+import { Color } from '../utils/StyleUtils';
+
+export const startPrompt = {
+  text: 'Start the quiz?',
+  options: ['Yes', 'No']
+};
+
+export const questionPrompt = 'What is the correct answer?';
```

Comment:
Would be great to allow the storywriter to customize this prompt - would this be difficult to implement?

(I recall we also briefly discussed this in person - we can simply open a new issue if this is not trivial)

Reply:
I think this may require to update the syntax rule for quiz txt writing and the parser. Any other thoughts about this issue? @reginateh

Reply:
Is it the prompt at line 11 you are referring to? Shouldn't be hard to update the syntax and parser I think.

Reply:
Can just add a new line for each question in the txt file and replace the default text with it.